### DISCUSSION TITLE
Is there a way to change the attribution position for Mapbox?

### DISCUSSION THREAD

**Discussion by 1mike12** [rnmapbox/maps#1885] [Q&A] [answered]
The defaults for ios and android are different. I would like the little (i) icon on the iOS version to also be on the left, since I have buttons I want to place on the bottom right. 

The online docs say that moving it around is possible 
>By default, the information button is located on the bottom right of the map. You may move the information button to a different position within the map view. For display options, see the API documentation for the [AttributionButtonOptions](https://docs.mapbox.com/ios/maps/api/10.0.1/Structs/AttributionButtonOptions.html).

https://docs.mapbox.com/help/getting-started/attribution/

<img width="841" alt="Screen Shot 2022-05-04 at 3 06 00 PM" src="https://user-images.githubusercontent.com/4983282/166687332-977057e8-0db4-4f66-91eb-729c09227a38.png">

**Comment by Cmoen11** [comment, marked as answer]
Hi 

We're doing it like this:

```
attributionPosition={
	Platform.OS === "ios"
		? {
			top: undefined,
			left: 100,
			bottom: 7,
			right: undefined,
		}
		: undefined
}
```

**Reply by 1mike12** [reply]
thanks!
